arthur_bench/scoring/exact_match.py

```diff
@@ -16,6 +16,14 @@ def __init__(self, case_sensitive=True):
     def name() -> str:
         return "exact_match"
 
+    @staticmethod
+    def is_categorical() -> bool:
+        return True
+
+    @staticmethod
+    def categories() -> Optional[List[float]]:
```

Comment:
these type hints for each of these is a bit odd. Its not optional if its hardcoded to return? I get its a base class thing but maybe the classes returning None should throw instead